.gitignore

```diff
@@ -2,7 +2,7 @@ target/
 *~
 TODO.txt
 *-gh/
-*.osm
+*.osm*
```

Comment:
This is a bit dangerous as we have many *osm.gz files under core/files/ ... probably better to create a folder like osm-data (and put the temp osm files there?) which is then ignored?